Samples/Samples/View/HaptickFeadbackPage.xaml.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Samples.View
+{
+    public partial class HaptickFeadbackPage : BasePage
```

Comment:
```suggestion
    public partial class HaptickFeedbackPage : BasePage
```

typo :wink:  (here and there ...)